app/javascript/src/QuickStarts/components/CustomCatalog.tsx

```diff
@@ -25,80 +25,59 @@ import './QuickStartContainer.scss'
-                    <QuickStartTile
-                      isActive={id === activeQuickStartID}
-                      quickStart={quickStart}
-                      status={getQuickStartStatus(allQuickStartStates!, id)}
-                    />
-                  </GalleryItem>
-                )
-              })}
-          </Gallery>
-        </QuickStartCatalogSection>
-        <QuickStartCatalogSection>
-          <TextContent>
-            <Text className="catalog" component="h2">Common instance for creating and tracking APIs</Text>
-          </TextContent>
-          <Gallery hasGutter className="pfext-quick-start-catalog__gallery">
-            {allQuickStarts!
-              .filter((quickStart: QuickStart) => quickStart.metadata.commonInstance)
-              .map((quickStart: QuickStart) => {
-                const {
-                  metadata: { name: id }
-                } = quickStart
-                return (
-                  <GalleryItem key={id} className="pfext-quick-start-catalog__gallery-item">
-                    <QuickStartTile
-                      isActive={id === activeQuickStartID}
-                      quickStart={quickStart}
-                      status={getQuickStartStatus(allQuickStartStates!, id)}
-                    />
-                  </GalleryItem>
-                )
-              })}
-          </Gallery>
-        </QuickStartCatalogSection>
-        <QuickStartCatalogSection>
-          <TextContent>
-            <Text className="catalog" component="h2">Basic API integration setup</Text>
-          </TextContent>
-          <Gallery hasGutter className="pfext-quick-start-catalog__gallery">
-            {allQuickStarts!
-              .filter((quickStart: QuickStart) => quickStart.metadata.basicAPIIntegrationSetup)
-              .map((quickStart: QuickStart) => {
-                const {
-                  metadata: { name: id }
-                } = quickStart
-                return (
-                  <GalleryItem key={id} className="pfext-quick-start-catalog__gallery-item">
-                    <QuickStartTile
-                      isActive={id === activeQuickStartID}
-                      quickStart={quickStart}
-                      status={getQuickStartStatus(allQuickStartStates!, id)}
-                    />
-                  </GalleryItem>
-                )
-              })}
-          </Gallery>
-        </QuickStartCatalogSection>
+        {categories.map(category => {
+          return (
+            <QuickStartCatalogSection key={category.id}>
+              <TextContent>
+                <Text className="catalog" component="h2">{category.title}</Text>
+              </TextContent>
+              <Gallery hasGutter className="pfext-quick-start-catalog__gallery">
+                {allQuickStarts!
+                  .filter((quickStart: QuickStart) => quickStart.metadata.labels?.category == category.id)
```

Comment:
So, I put the `category` under `labels`, because this is how the `QuickStart` type is defined, there is a "free-form" `labels` section (there is also `annotations`), but other fields are strictly defined, so this was causing a warning:
![Screenshot from 2023-12-21 12-55-57](https://github.com/3scale/porta/assets/1270328/5f9153e9-4731-454f-82d5-01a7806e30eb)